src/plone/restapi/deserializer/blocks.py

```diff
@@ -104,20 +109,22 @@ def __call__(self, block):
 class TextBlockDeserializer(object):
     order = 100
     block_type = "text"
+    disabled = os.environ.get("disable_transform_resolveuid", False)
 
     def __init__(self, context, request):
         self.context = context
         self.request = request
 
     def __call__(self, block):
         # Convert absolute links to resolveuid
-        # Assumes in-place mutations
-
+        #   http://localhost:55001/plone/link-target
+        #   ->
+        #   ../resolveuid/023c61b44e194652804d05a15dc126f4
         entity_map = block.get("text", {}).get("entityMap", {})
         for entity in entity_map.values():
             if entity.get("type") == "LINK":
-                href = entity.get("data", {}).get("href", "")
```

Comment:
@sneridagh we really have to properly document when to user href and when to use URL. This won't be the last time we mix things up.